### PR TITLE
Fix/delete to delete folder

### PR DESCRIPTION
**Pull request by AdrienRem**

### CODE REVIEW

**Review by lucieho — APPROVED** [review]
Nice refactoring!
Just making sure: I understood that from now on positions are not uploaded as map in History (Firebase)?

**Comment by AdrienRem** [comment]
> Nice refactoring!
> Just making sure: I understood that from now on positions are not uploaded as map in History (Firebase)?

Lucas is working on that but it's not merged yet

File: app/src/androidTest/java/ch/epfl/sdp/map/fragment/PathHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+        if (!mapFragment.getPathsHandler().getYesterdayPathCoordinates().isEmpty()) {
+            testLayerIsSet(YESTERDAY_PATH_LAYER_ID);
+            pathCoordIsEmpty = false;
+        } else {
+            pathCoordIsEmpty = true;
+        }
+    }
+
+    @Test(timeout = 20000)
```
[the remark this answers is not in the document]
Comment (TheBlueHawk) [reply app/src/androidTest/java/ch/epfl/sdp/map/fragment/PathHandlerTest.java:106]: Too many methods calls to be refactored in a cleaner way

**Review by TheBlueHawk — APPROVED** [review]
Good work, I'll just push a quick additional refactor to fix a code duplication in testPathsHandler

**Pull request merged**